scripts/sanity_chk/harness.py

```diff
@@ -121,6 +127,40 @@ def handle(self, line):
 
         self.process_test(line)
 
+class Pytest(Harness):
+    def handle(self, line):
+        cmd = 'pytest -s ' +  self.handle_script + \
```

Comment:
populate this as a list, it will be more readable and manageable.

Reply:
fixed,